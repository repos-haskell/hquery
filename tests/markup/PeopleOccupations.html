<div class="person">
  <div class="name"></div>
  <div class="occupation"></div>
</div>
