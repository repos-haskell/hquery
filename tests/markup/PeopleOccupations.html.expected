<div class="person">
  <div class="name">Bob</div>
  <div class="occupation">Engineer</div>
</div>
<div class="person">
  <div class="name">Sally</div>
  <div class="occupation">CEO</div>
</div>
<div class="person">
  <div class="name">Rutherford</div>
  <div class="occupation">Hacker</div>
</div>
<div class="person">
  <div class="name">Vikki</div>
  <div class="occupation">Hybrid</div>
</div>
